worlds/musedash/MuseDashData.txt

```diff
@@ -537,4 +537,11 @@ Ruler Of My Heart VIVINOS|71-1|Valentine Stage|False|2|4|6|
 Reality Show|71-2|Valentine Stage|False|5|7|10|
 SIG feat.Tobokegao|71-3|Valentine Stage|True|3|6|8|
 Rose Love|71-4|Valentine Stage|True|2|4|7|
-Euphoria|71-5|Valentine Stage|True|1|3|6|
\ No newline at end of file
+Euphoria|71-5|Valentine Stage|True|1|3|6|
```

Comment:
Would there be issues ending this file with a newline? Would lead to a cleaner diff on future updates.

Reply:
Looks like there isn't. `.splitlines()` seems to remove the trailing line properly.